rover_msgs/Obstacle.lcm

```diff
@@ -3,4 +3,5 @@ package rover_msgs;
 struct Obstacle {
 	boolean detected;
 	double bearing; // from straight ahead	
+	double distance; // from straight ahead
```

Comment:
It's concerning that we had to add this considering CV was already planning on sending us a distance, we should make sure this is what they are using.